cms/djangoapps/contentstore/features/advanced_settings.py

```diff
@@ -32,7 +32,7 @@ def i_select_advanced_settings(step):
     world.visit(link)
 
 
-@step('I am on the Advanced Course Settings page in Studio$')
+@step('I am on Advanced Course Settings page in Studio$')
```

Comment:
Can you revert the change you have made here, it is not required and it may be the cause of an error if lettuce use this text to search for the step